man/systemd-soft-reboot.service.xml

```diff
@@ -116,6 +117,7 @@ DefaultDependencies=no
 After=basic.target
 Conflicts=reboot.target kexec.target poweroff.target halt.target rescue.target emergency.target
 Before=shutdown.target rescue.target emergency.target
+BindReadOnlyPaths=/run/systemd/journal/socket /run/systemd/journal/stdout
```

Comment:
Hmm, this should be placed in [Service]?
Also, I think #32487 is a better choice here.

Reply:
whops

Reply:
that PR won't be merged for 256, so I think this hint should go in now, and we can revise in the next release